<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceClientClass</class>
 <widget class="QMainWindow" name="DeviceClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>911</width>
    <height>455</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>911</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DeviceClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Запись</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Группа 1                                                                          Группа 2                                                                        Группа 3                                                  </string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <row>
          <property name="text">
           <string>РГ0</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ7</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Данные hex</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Комментарии</string>
          </property>
         </column>
         <item row="2" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <row>
          <property name="text">
           <string>РГ8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ9</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ10</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ11</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ12</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ13</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ14</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ15</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ16</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ17</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ18</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ19</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ20</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ21</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ22</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ23</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ24</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ25</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ26</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ27</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ28</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ29</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ30</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ31</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Данные hex</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Комментарии</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="12" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="12" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="13" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="14" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="15" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="16" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="17" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="18" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="19" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="20" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="21" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="22" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="23" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <row>
          <property name="text">
           <string>РГ32</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ33</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ34</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ35</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ36</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ37</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ38</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>РГ39</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Данные hex</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Комментарии</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="7" column="1">
          <property name="text">
           <string>РЕЗЕРВ</string>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Чтение</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>ALL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Группа 1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Группа 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Группа 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DeviceClient.qrc"/>
 </resources>
 <connections/>
</ui>
